docs/Contributing/API-for-contributors.md

```diff
@@ -668,19 +672,212 @@ Content-Type: application/octet-stream
 
 `Status: 204`
 
+### Update ABM token's teams
+
+`PATCH /api/v1/fleet/abm_tokens/:id/teams`
+
+#### Parameters
+
+| Name | Type | In | Description |
+| ---- | ---- | -- | ----------- |
+| id | integer | path | *Required* The ABM token's ID |
+| macos_team_id | integer | body | macOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager. If not specified, defaults to "No team" |
+| ios_team_id | integer | body | iOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager. If not specified, defaults to "No team" |
+| ipados_team_id | integer | body | iPadOS hosts are automatically added to this team in Fleet when they appear in Apple Business Manager. If not specified, defaults to "No team" |
+
+#### Example
+
+`PATCH /api/v1/fleet/abm_tokens/1/teams`
+
+##### Request body
+
+```json
+{
+  "macos_team_id": 1,
+  "ios_team_id": 2,
+  "ipados_team_id": 3,
+}
+```
+
+##### Default response
+
+`Status: 200`
+
+```json
+{}
+```
+
+### Renew ABM token
+
+`PATCH /api/v1/fleet/abm_tokens/:id/renew`
```

Comment:
Hey @roperzh I'm assuming we're going to have to have an endpoint to renew a specific ABM (or VPP) token.

Why? Because we'll need to keep track of the private key for each of these tokens.

Does that look/sound right?

Reply:
@noahtalerman that sounds good, alternatively if they're bound to a team we could use the team id, but this looks good too 👍